ops/terraform/modules/resources/bfd_pipeline/main.tf

```diff
@@ -134,6 +134,35 @@ resource "aws_security_group_rule" "allow_db_primary_access" {
   source_security_group_id = aws_security_group.app.id # The EC2 instance for the BFD Pipeline app.
 }
 
+# NACL to manage communication between the BFD and RDA environments
+# By default this NACL won't add any additonal security, however if for any reason
+# we need to shut off communication between the two VPCs, one could simply flip the
+# action(s) defined in these rules from "ALLOW" to "DENY"
+resource "aws_network_acl" "rda" {
+  count = var.mpm_rda_cidr_block ? 1 : 0
+
+  vpc_id      = var.env_config.vpc_id
+  tags        = merge({ Name = "bfd-${var.env_config.env}-etl-app" }, var.env_config.tags)
+
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    rule_no     = 100
+    action      = "allow"
+    protocol    = "all"
+    cidr_block  = var.mpm_rda_cidr_block
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    rule_no     = 200
```

Comment:
Should we also restrict the outbound port to 443?

Reply:
Thats an interesting idea, we could switch this NACL to use explicit "deny" actions, then create an additional NACL with a lower rule order that "allows" communication over 443.  I think its a good idea, but would love to hear more from the BFD engineers before making this more complex.  Maybe that level of control with another "trusted" VPC isn't necessary?

Reply:
Ideally yes. But we can harden it later if you prefer.

Reply:
@keithdadkins I took a stab at refactoring this into 2 rules, let me know what you think